### PR TITLE
[IMP] mail: bigger font size for messages that only contains emoji 

### PR DESCRIPTION
**Pull request by tsm-odoo**
This PR increases the size of the message body when the later only
contains emojis.

task-3651454

### CODE REVIEW

File: addons/mail/static/src/core/common/message_model.js [hunk 1]
```diff
@@ -62,6 +62,18 @@ export class Message extends Record {
     /** @type {string} */
     scheduledDatetime;
     starredPersonas = Record.many("Persona");
+    onlyContainsEmojis = Record.attr(false, {
```
Comment (alexkuhn) [inline addons/mail/static/src/core/common/message_model.js:65]: Nitpicking: could be named more briefly, e.g. `onlyEmojis`

File: addons/mail/static/src/core/common/message_model.js [hunk 2]
```diff
@@ -62,6 +62,18 @@ export class Message extends Record {
...
+        compute() {
+            const div = document.createElement("div");
+            div.innerHTML = this.body;
+            const bodyWithoutTags = div.textContent;
+            const withoutEmojis = bodyWithoutTags.replace(
+                /\p{Emoji_Presentation}|\p{Emoji}\uFE0F|\u200d/gu,
```
Comment (phenix-factory) [inline addons/mail/static/src/core/common/message_model.js:71]: There are multiple instances of this regexp in the code:
* https://github.com/odoo/odoo/blob/master/addons/mail/static/src/core/common/thread_service.js#L615
* https://github.com/odoo/odoo/blob/master/addons/mail/static/src/js/emojis_mixin.js#L20

Maybe it should be an exported `const`?

File: addons/mail/static/tests/legacy/message/message_tests.js [hunk 3]
```diff
@@ -1689,3 +1689,36 @@ QUnit.test("Click on view reactions shows the reactions on the message", async (
...
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({
+        channel_type: "channel",
+        name: "channel1",
+    });
+    const { openDiscuss, openFormView } = await start();
+    await openDiscuss(channelId);
+    await insertText(".o-mail-Composer-input", "🥳");
+    await click(".o-mail-Composer-send:enabled");
+    await openFormView("res.partner", pyEnv.currentPartnerId);
```
Comment (alexkuhn) [inline addons/mail/static/tests/legacy/message/message_tests.js:1719]: The steps related to `openDiscuss()` in the "chatter" test look irrelevant

**Review by alexkuhn — COMMENTED** [review]
LGTM otherwise

File: addons/mail/static/tests/legacy/message/message_tests.js (hunk 3 above)
Comment (tsm-odoo) [reply addons/mail/static/tests/legacy/message/message_tests.js]: it's better to:
- Open discuss and check that the emoji is bigger
- Open the chatter then check its not the case
Otherwise we would have a negative check only which is not reliable

**Comment by tsm-odoo** [comment]
@alexkuhn updated to make the tests more robust as discussed :+1:

**Review by alexkuhn — APPROVED** [review]
@robodoo override=ci/security
sanitized message body so ok to `innerHTML=`

**Review by alexkuhn — APPROVED** [review]
@robodoo r+ delegate+

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

**Comment by tsm-odoo** [comment]
@robodoo r+

:arrow_up: pushed to remove tests relative to recent emojis being updated after posting a message containing its shortcode (which was removed by this PR as it was unreliable)
